webpack/assets.js

```diff
@@ -0,0 +1,42 @@
+import WebpackIsomorphicToolsPlugin from 'webpack-isomorphic-tools/plugin';
```

Comment:
Can do destructuring with this import? It would save as the extra long lines in the definition.

```js
import {url_loader_parser, css_modules_loader_parser, style_loader_path_extractor} from 'webpack-isomorphic-tools/plugin'
```